website/docs/index.html.markdown

```diff
@@ -38,14 +38,14 @@ Use the navigation to the left to read about the available resources.
 
 The following arguments are supported.
 
-| Argument               | Required? | Description                                                                                                                                                                 |
-| ---------------------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `account_id`           | Required  | Your New Relic account ID. The `NEW_RELIC_ACCOUNT_ID` environment variable can also be used.                                                                                |
-| `api_key`              | Required  | Your New Relic Personal API key (usually prefixed with `NRAK`). The `NEW_RELIC_API_KEY` environment variable can also be used.                                              |
-| `region`               | Required  | The region for the data center for which your New Relic account is configured. The `NEW_RELIC_REGION` environment variable can also be used. Valid values are `US` or `EU`. |
-| `insecure_skip_verify` | Optional  | Trust self-signed SSL certificates. If omitted, the `NEW_RELIC_API_SKIP_VERIFY` environment variable is used.                                                               |
-| `insights_insert_key`  | Optional  | Your Insights insert key used when inserting Insights events via the `newrelic_insights_event` resource. Can also use `NEW_RELIC_INSIGHTS_INSERT_KEY` environment variable. |
-| `cacert_file`          | Optional  | A path to a PEM-encoded certificate authority used to verify the remote agent's certificate. The `NEW_RELIC_API_CACERT` environment variable can also be used.              |
+| Argument               | Required? | Description                                                                                                                                                                                        |
+| ---------------------- | --------- |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `account_id`           | Required  | Your New Relic account ID. The `NEW_RELIC_ACCOUNT_ID` environment variable can also be used.                                                                                                       |
+| `api_key`              | Required  | Your New Relic Personal API key (usually prefixed with `NRAK`). The `NEW_RELIC_API_KEY` environment variable can also be used.                                                                     |
+| `region`               | Required  | The region for the data center for which your New Relic account is configured. The `NEW_RELIC_REGION` environment variable can also be used. Valid values are `US` or `EU`. Default value is `US`. |
```

Comment:
I think it's not required? It's a bit odd, since it defaults to `US` if you don't specific it. So it's required in code, but if you don't set it automatically defaults to 'US'

Reply:
Good call. Technically it's optional since we automatically default to `US`.

Reply:
✅